syntax = "proto2";

package interface.playback;

message PlaybackPause {
}

message PlaybackTerminate {
}

message PlaybackNextIteration {
}

message PlaybackCommand {
    oneof command {
        double playback_speed_x = 5;
        PlaybackPause playback_pause = 4;
        PlaybackNextIteration playback_next_iteration = 6;
        PlaybackTerminate playback_terminate = 7;
    }
}
